docs/tutorials.rst

```diff
@@ -0,0 +1,562 @@
+
+In the previous tutorials we have always specified the target BIDS directory as an explicit path. It is also possible
+to use the current working directory as the BIDS directory, provided that it is either empty or already a partial BIDS
+dataset as defined by a ``dataset_description.json`` file with a ``BIDSVersion`` value specified within.
+
+To test this out, we can create a new empty directory and navigate into it before converting:
+
+.. tabs::
+    .. tab:: CLI
+
+        .. tabs::
+            .. tab:: Unix / macOS
+
+                .. code-block:: bash
+
+                    mkdir ~/.nwb2bids/tutorials/ephys_tutorial_dataset/bids_dataset
+                    cd ~/.nwb2bids/tutorials/ephys_tutorial_dataset/bids_dataset
+
+                    nwb2bids convert \
+                        ~/.nwb2bids/tutorials/ephys_tutorial_dataset/ephys_session_3.nwb \
+                        ~/.nwb2bids/tutorials/ephys_tutorial_dataset/some_sessions
+
+            .. tab:: Windows
+
+                .. code-block:: bash
+
+                    mkdir ~/.nwb2bids/tutorials/ephys_tutorial_dataset/bids_dataset
+                    cd ~/.nwb2bids/tutorials/ephys_tutorial_dataset/bids_dataset
+
+                    nwb2bids convert ^
+                        ~/.nwb2bids/tutorials/ephys_tutorial_dataset/ephys_session_3.nwb ^
+                        ~/.nwb2bids/tutorials/ephys_tutorial_dataset/some_sessions
+
+        The command line can take any number of inputs (separated by spaces) prior to other flags such as
+        ``--bids-directory``. These inputs can be any mix of files or directories.
+
+    .. tab:: Python Library
+
+        .. code-block:: python
+
+            >>> import os
+            >>> import pathlib
+            >>>
+            >>> import nwb2bids
+            >>>
+            >>> tutorial_directory = pathlib.Path.home() / ".nwb2bids/tutorials/ephys_tutorial_dataset"
+            >>> nwb_paths = [
+            ...     tutorial_directory / "ephys_session_3.nwb",
+            ...     tutorial_directory / "some_sessions",
+            ... ]
+            >>> bids_directory = tutorial_directory / "implicit_bids_dataset"
+            >>> bids_directory.mkdir(exist_ok=True)
+            >>> os.chdir(path=bids_directory)
+            >>>
+            >>> notifications = nwb2bids.convert_nwb_dataset(nwb_paths=nwb_paths)
+
+And the results should match what we saw at the end of :ref:`tutorial-multiple-inputs`.
+
+
+
+.. _tutorial-additional-metadata:
+
+Tutorial 5 - Additional metadata
+---------------------------------
+
+NWB files don't always include all the little metadata details you might want to include in BIDS. To include manual
+metadata that is unable to be extracted from the source files, you can provide additional metadata through a JSON
+structure that mirrors the BIDS specification nested under subject and session identifiers (which should match the
+``subject_id`` and ``session_id`` found within each file).
```

Comment:
this is unclear to me and no example below? moreover note says that it seems only this dataset_description is supported ATM.  Should we remove this for now?

Reply:
Yes, I will remove

Reply:
Done